bcda/testUtils/utils.go

```diff
@@ -20,21 +20,6 @@ func PrintSeparator() {
 	fmt.Println("**********************************************************************************")
 }
 
-func CreateStaging(jobID string) {
```

Comment:
Why was this removed?

Reply:
Not sure why the comment fell off, but we only used this function in main_test.go. With the refactoring of the test setup, we no longer needed a dedicated function for this:

https://github.com/CMSgov/bcda-app/pull/576#discussion_r516309340